nanoc-dart-sass/lib/nanoc/dart_sass/filter.rb

```diff
@@ -31,7 +31,7 @@ def initialize(items, source_item)
           @source_item = source_item
         end
 
-        def canonicalize(url, **)
+        def canonicalize(url, *, **)
```

Comment:
- `sass-embedded < 1.68.0`: `def canonicalize(url, from_import:)`
- `sass-embedded >= 1.68.0`: `def canonicalize(url, canonicalize_context)`

Since we don't use the context, we can define this method as `def canonicalize(url, *, **)`, which is compatible with both all past sass-embedded versions.